src/annotator/test/guest-test.coffee

```diff
@@ -1,11 +1,12 @@
-Annotator = require('annotator')
 proxyquire = require('proxyquire')
 
 adder = require('../adder')
 Observable = require('../util/observable').Observable
+Plugin = require('../plugin')
 
-$ = Annotator.$
-Annotator['@noCallThru'] = true;
+Delegator = require('../delegator')
+$ = require('jquery')
+Delegator['@noCallThru'] = true
```

Comment:
The addition of `@noCallThru` here will modify the real `Delegator` module as seen by any other tests which use it. This is OK for the moment since the existing code was already doing this, but in general it is something to avoid.